tests/test_generic/tests/test_industry_requirements.py

```diff
@@ -70,16 +70,16 @@ def test_knowledge_article_notification(self):
             )
 
     def test_cloc_exclude_view(self):
-        if not sys.argv[sys.argv.index('-d') + 1].endswith('imported_no_demo'):
-            return
         c = cloc.Cloc()
         c.count_database(self.env.cr.dbname)
         c.report(True)  # show details of cloc in the logs
         for module in self.installed_modules:
             for cloc_entry in c.modules.get(module, {}):
                 message = "The view '%s' is counted in the maintenance lines. " % cloc_entry
-                message += "Please add a '__cloc_exclude__' entry in 'ir_model_data'."
+                message += "Please add a '__cloc_exclude__' entry in 'manifest."
                 self.assertEqual(len(cloc_entry.split(':')), 2, message)
+                file_path = cloc_entry.split(':')[0]  # Extract the file path from the cloc entry
+                self.assertTrue(os.path.exists(file_path), "File '%s' does not exist." % file_path)
 
     def test_sale_ok_and_is_published_in_db(self):
```

Comment:
I am not a big fan of this. The cloc_exclude in the manifest exclude everything in the file. However everything should maybe not be excluded. Also is the test that the file exist revelant ?
There is already a test that all the files in the manifest exist and vice-versa.

Reply:
Excluding xmlids that don't count in the cloc is fine. But it creates new xmlid for the same record, which is not necessary, so I kind of agree with you.
About the test, I specifically asked for one because the current test was failing to detect that an entry in cloc_exclude did not exist. So I'd go for a test checking that, but maybe not in this file.

Reply:
where did this test go? Check that what is inside `cloc_exclude` in the manifest is an existing file

Reply:
I still think the test is required and would help

Reply:
(https://github.com/odoo/industry/pull/296) All the tests cases are being fixed in this PR so IMO there is no need to add it here again as it is already done in this PR